packages/types/src/UnixTime.ts

```diff
@@ -115,4 +118,15 @@ export class UnixTime {
   toYYYYMMDD() {
     return this.toDate().toISOString().slice(0, 10)
   }
+
+  static isNumberUnixTimestamp(timestamp: number): boolean {
```

Comment:
I find this method quite weird. Its purpose is to guard against constructor throwing, but it uses different logic than the constructor.
I would much prefer it to have the exact same logic as the constructor and be renamed to `isSafeToCast`. That also means that the `YEAR_2019_TIMESTAMP` constant should be removed.